src/components/Navbar.jsx

```diff
@@ -2,8 +2,38 @@ import React from "react";
 import { NavLink } from "react-router-dom";
 import { FaPaperPlane } from "react-icons/fa";
 
+function Navbar() {
+    return(
+        <NavWrapper className="navbar navbar-expand-sm navbar-light px-sm-5">  
+            <div className="ml-auto navbar-nav">
+                <Link to="/">
+                    <div className="nav-item text-dark px-4 font-weight-bold">
+                        Home
+                    </div>
+                </Link>
+                <Link to="/about">
+                    <div className="nav-item text-dark px-4 font-weight-bold">
+                        About
+                    </div>
+                </Link>
+                <Link to="/contact">
+                    <div className="nav-item text-dark px-4 font-weight-bold">
+                        Contact
+                    </div>
+                </Link>
+                <Link to="/detail">
+                    <div className="nav-item text-dark px-4 font-weight-bold">
+                        Contributers
+                    </div>
+                </Link>
+            </div>            
+        </NavWrapper>
+    )
+}
+
 const Navbar = () => {
 
+
   /*-------All Nav links goes here------------------ */
   const Navlinks = [
     { name: "Home", path: "/" },
```

Comment:
This Navbar is well Written and can be kept instead of the one of top . So delete the top one!